audio/channel_receive.cc

```diff
@@ -638,8 +638,8 @@ void ChannelReceive::ReceivePacket(const uint8_t* packet,
     payload = decrypted_audio_payload.data();
     payload_data_length = decrypted_audio_payload.size();
   } else if (crypto_options_.sframe.require_frame_encryption) {
-    RTC_DLOG(LS_ERROR)
-        << "FrameDecryptor required but not set, dropping packet";
+    // RTC_DLOG(LS_ERROR)
+    //     << "FrameDecryptor required but not set, dropping packet";
```

Comment:
This is annoying if you received media in between when you create an RtpReceiver and set the Decryptor, which is very short but can happen.